src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx

```diff
@@ -25,6 +25,11 @@ export type SurveyOptionsViewCell =
     }[]
   | null;
 
+const surveyOptionsColumnSortRank = (str: string) => {
+  const split = str.split('|');
+  return -split[split.length - 1].length;
+};
```

Comment:
Same here with making it more self-documenting with saving and naming step by step - it is so helpful for the future when event small utility functions like this have code in them that is very clear and named step by step. 

I'm thinking something like (assuming I have correctly understood what the code does, so please add your own if I've messed up here): 

```
const surveyOptionsColumnSortRank = (allOptionsJoined: string) => {
   const options = allOptionsJoined.split('|');
   const lastOption = options[options.length - 1];
   return -lastOption.length;
}
```